tests/phpunit/CRM/Mailing/BaseMailingSystemTest.php

```diff
@@ -57,7 +55,7 @@
   public function setUp() {
     $this->useTransaction();
     parent::setUp();
-    CRM_Mailing_BAO_MailingJob::$mailsProcessed = 0; // DGW
```

Comment:
For the curious, `DGW` stands for "*Dear God, Why... Why do I have to reset this silly static variable by hand?*"

Reply:
:-)